packages/frontend/src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
-import './App.css';
-import { Link } from 'react-router-dom';
+import Nav from './components/landingNav';
+import Describe from './sections/toolDescribe';
+import Features from './sections/features';
+import './index.css';
 import { signOut } from 'aws-amplify/auth';
 
```

Comment:
Notice how you are importing the `landingNav` but you are using it as `Nav`, this is confusing.